views/index.ejs

```diff
@@ -169,6 +169,9 @@
     <div class = "dropDownItemDiv" id="editSite" >
     <img src="images/blue-dot.png">Edit my site</div>
   <div class = "dropDownItemDiv" id="newSite"><img src="images/blue-dot.png">Add new site</div>
+  <% if (user.admin) { %>
+    <a href="/admin"><div class = "dropDownItemDiv" id="adminPage"><span class="glyphicon glyphicon-user"></span>Admin Page</div></a>
+  <% } %>
```

Comment:
Please move this to the top of the dropdown list for admins (we'll assume Atlas admins will be more likely to be visiting Atlas for administration than managing their markers).

Reply:
Yup.. and its a bit too close to the logout button. Pushed the change :)